example/optsched-cfg/sched.ini

```diff
@@ -12,7 +12,8 @@ PRINT_SPILL_COUNTS YES
 # First pass minimizes RP and second pass tries to balances RP and ILP.
```

Comment:
Why are all these changes in this sched.ini? Are we checking in something that we verified as the current best setup or something like that?

Reply:
Yes -- this is the sched.ini to produce baseline execution results in the new ROCm environment